modules/ballerina-core/src/main/java/org/ballerinalang/util/codegen/AnnAttributeValue.java

```diff
@@ -31,6 +31,11 @@
     private double floatValue;
     private String stringValue;
     private boolean booleanValue;
+    private boolean runTimeProvisioned = false;
```

Comment:
i felt i may mislead since basic literals is also a kind of constant, shall i change?